minard/templates/burst_run_detail.html

```diff
@@ -0,0 +1,134 @@
+{% extends "layout.html" %}
+{% block title %}Burst Run Detail{% endblock %}
+{% block head %}
+{{ super() }}
+<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/pca.css') }}" media="screen">
+{% endblock %}
+{% block body %}
+{{ super() }}
+
+
+<div class="container">
+  {{ subrun }}
+  {{ sub }}
+<h2>Run {{ data["run"] }} details</h2>
```

Comment:
Same comment as above. Should indent things here too. See https://github.com/snoplus/minard/blob/master/minard/templates/snostream.html for an example.